### PR TITLE
Reword the 'features' page

### PR DESCRIPTION
**Pull request by yuvipanda**
- Remove 'distributing content', add 'authentication' instead
- Trim content size

Part of https://github.com/2i2c-org/2i2c-org.github.io/pull/42

### CODE REVIEW

File: content/infrastructure/features.md [hunk 1]
```diff
@@ -11,36 +11,36 @@ weight = 10
...
+  name = "Custom environments"
+  description = "Python, R, Julia - use whatever custom data science stack you want."
 
 [[feature]]
   icon = "desktop"
   icon_pack = "fas"
-  name = "Provide a variety of open source interfaces"
-  description = "Use a variety of user interfaces, including Jupyter Notebook, Jupyter Lab, RStudio, VSCode. If it can be run in a browser, it can run on a 2i2c Hub."
+  name = "Data Science UI"
+  description = "JupyterLab, Jupyter Notebook, RStudio - pick tools that fits your workflow."
```
Comment (choldgraf) [inline content/infrastructure/features.md]:
```suggestion
  description = "JupyterLab, Jupyter Notebook, RStudio - pick tools that fit your workflow."
```

**Comment by choldgraf** [comment]
this is great, one minor comment but I'm +1 other than that

**Comment by yuvipanda** [comment]
@choldgraf fixed!

**Comment by choldgraf** [comment]
merged!

**Pull request merged**